qutip/solve/steadystate.py

```diff
@@ -449,7 +462,10 @@ def _steadystate_LU_liouvillian(L, ss_args, has_mkl=0):
         perm2 = scipy.sparse.csgraph.reverse_cuthill_mckee(L)
         _rcm_end = time.time()
         rev_perm = np.argsort(perm2)
+        print(np.round(L.toarray(),3))
         L = _permute(L, perm2, perm2)
+        print(np.round(L.toarray(),3))
+        print()
```

Comment:
Are these print statements supposed to be there¿

Reply:
weirdly enough, most of the errors disappear if only this file is merged into dev.major. Except for the two related to iterative methods not converging. It also seems to be way slower once only this file is merged.

Reply:
It seems I was not runningthe tests that do include 'use_rcm'. We do have to remove the print statements though.

Reply:
This is definitely where the error occurs. I compared the output of `sp_permute` (version in master) with `_permute` and they do not return the same output for the same matrices and permutation indices. I will include a fix but lets talk about it later in the meeting.